x-pack/plugins/security/public/management/table_utils.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { css } from '@emotion/react';
+import type { ReactNode } from 'react';
+import React from 'react';
+
+interface ActionsEuiTableFormattingProps {
+  children: ReactNode;
+}
+export const ActionsEuiTableFormatting = React.memo<ActionsEuiTableFormattingProps>(
+  ({ children }) => (
+    <div
+      css={css`
+        .euiTableRowCell--hasActions .euiButtonEmpty .euiButtonContent {
+          padding: 0px 0px;
+          .euiButtonEmpty__text {
+            display: none;
+          }
+        }
+      `}
+    >
+      {children}
+    </div>
+  )
+);
```

Comment:
Sorry just one more nit :) 
You have good reasoning for this in https://github.com/elastic/kibana/pull/118434/files#r749691551, but that's not immediately apparent in the code. Can you add a comment explaining why this is needed, so we can save our future selves some confusion?

Reply:
it is to match the design from EUI and only have the icon to show when you hover the row like in the EUI demo page

Reply:
fix with the latest commit